subprojects/docs/src/docs/userguide/img/taskInputsOutputs.graffle

```diff
@@ -0,0 +1,816 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe we could put the original image files (like Photoshop, Graffle etc.) into a separate directory. So far we didn't check in those files but we should definitely keep them version controlled.